tests/pytest.ini

```
@@ -1,4 +1,6 @@
 [pytest]
+norecursedirs = ptftests acstests
+addopts = -ra --allow_recover --showlocals --module-path ../ansible/library/ --show-capture=no --ignore=ptftests --ignore=acstests --ignore=test_vrf.py
```

What happens if I want to disable one of these options, like `--showlocals` for instance?

---

do we need ignore=...tests since there are not being traversed by the above norecursedirs options?

---

@tahmed-dev yes, please see the PR description. norecursedir only suppress error, but didn't prevent test case collection.

@daall I'll get back to you, need to investigate.

---

Checked the available pytest CLI options, there is no way to override `--showlocals` from CLI. The `--showlocals` argument has action `store_true`. We can't specify some like `--showlocals=False`.

---

Thanks folks, removed this one and --allow-recover